### PR TITLE
Adding data on Alan's outreachy internship

### PR DESCRIPTION
**Pull request by panglesd**
This is a PR on #2689's branch to add @yokurang's data on his outreachy internship.

CC @Julow.

### CODE REVIEW

File: data/outreachy.yml [hunk 1]
```diff
@@ -429,4 +429,31 @@ rounds:
...
+
+          The goal of this project is to develop a visualization tool on top of
+          Git diff. The visualization tool is a terminal UI, and the main task
+          of the internship is to make it highly accessible to screen readers.
+        mentee: "Alan Matthew"
+        source: https://github.com/panglesd/diffcessible
+        mentors:
+          - Paul-Elliot Anglès d'Auriac <peada@free.fr>
+          - Jules Aguillon <juloo.dsi@gmail.com>
+        video: https://watch.ocaml.org/w/peT3MdWjS1BYYMbowEJ1gv?start=2m53s
```
Comment (Julow) [inline data/outreachy.yml]: Maybe `3m07s` to skip over the setup?

**Review by Julow — APPROVED** [review]
Thanks! :)

**Comment by sabine** [comment]
that's weird, merging the other PR closed this branch. Sorry about that

**Comment by panglesd** [comment]
That PR was on #2689's branch (that is, `outreachy-summer-2024`). merging #2689 deleted the branch, which closed the PR (the PR was targeting a deleted branch.

Reopening and merging recreated the `outreachy-summer-2024` branch. The content of this PR is thus not on `main` yet. I'll open a PR on `main`.

**Comment by panglesd** [comment]
My bad, it is on `main`! I am confused, but that's not a problem!

**Comment by sabine** [comment]
oh yes, I merged and then opened a PR on main :slightly_smiling_face:

**Pull request merged**